client/components/async-load/index.jsx

```diff
@@ -6,8 +6,12 @@ import { omit } from 'lodash';
 
 export default class AsyncLoad extends Component {
 	static propTypes = {
+		placeholder: PropTypes.node,
 		require: PropTypes.func.isRequired,
-		placeholder: PropTypes.node
+	};
+
+	static defaultProps = {
+		placeholder: <div className="async-load__placeholder async-load" />,
```

Comment:
We should use `async-load__placeholder` only to make Eslint happy. We can open a follow up PR to cleanup that later.